chain/client/src/client_actor.rs

```diff
@@ -721,6 +722,30 @@ impl Handler<Status> for ClientActor {
         } else {
             None
         };
+        // Provide info about the current state of sync
+        let debug_sync_info = if msg.get_debug_sync_info {
+            Some(DebugSyncInfo {
+                network_info: NetworkInfoView {
+                    peer_max_count: self.network_info.peer_max_count,
```

Comment:
Let's implement 
`impl From<PeerInfo> for PeerInfoView` and `impl From<NetworkInfo> for NetworkInfoView`, example: https://github.com/near/nearcore/blob/master/core/primitives/src/views.rs#L830-L861
Then you can simple call 
`network_info: self.network_info.into()`